File: APIdoc/python/tutorials/listFoldersPreviewFile.py
email, password, apiKey = "", "" ,""

sessionKey = pio_auth(email, password, apiKey)

accessRules = pio_list_access_rules(sessionKey)

folders = pio_list_folders(sessionKey, accessRules["RESULT"]["DATA"][2][1])["RESULT"]["DATA"]

for f in folders:
	if(f[2] == "FILE"):
		preview = pio_file_preview(sessionKey, f[16], f[0], f[1])
		print(preview)

File: APIdoc/php/tutorials/listFoldersPreviewFile.php
$email = "";
$password = "";
$apiKey = "";
$sessionKey = pio_auth($email, $password, $apiKey);

$rules = pio_list_access_rules($sessionKey)["RESULT"]["DATA"];
$folders = $pio_list_folders($sessionKey, $rules[2]);
foreach($folders as $f){	
	if($f[2] == "FILE"){		
		$preview = pio_file_preview(sessionKey, $f[16], $f[0], $f[1]);
		echo $preview;
	}
}